types/errors/abci.go

```diff
@@ -104,6 +104,17 @@ func QueryResult(err error) abci.ResponseQuery {
 	}
 }
 
+// QueryResultWithDebug returns a ResponseQuery from an error. It will try to parse ABCI
+// info from the error. It will use debugErrEncoder if debug parameter is true.
```

Comment:
```suggestion
// info from the error. It will use debugErrEncoder if debug parameter is true.
// Starting from v0.46, this function will be removed, and be replaced by `QueryResult`.
```